metricbeat/module/vsphere/datastore/datastore.go

```diff
@@ -41,7 +41,7 @@ func init() {
 }
 
 // MetricSet type defines all fields of the MetricSet.
-type MetricSet struct {
+type DsMetricSet struct {
```

Comment:
Can we use full-name instead?
```suggestion
type DataStoreMetricSet struct {
```